### PR TITLE
docs(changelog): changelog for v1.10.0

### PR DESCRIPTION
**Pull request by mynktl**
List of the PR added in changelog:
- v1.10.0-RC1
  - https://github.com/openebs/velero-plugin/pull/72
  - https://github.com/openebs/velero-plugin/pull/76
  - https://github.com/openebs/velero-plugin/pull/79

- v1.10.0-RC2
  - https://github.com/openebs/velero-plugin/pull/85

- v1.10.0
  - https://github.com/openebs/velero-plugin/pull/85
  - https://github.com/openebs/velero-plugin/pull/72
  - https://github.com/openebs/velero-plugin/pull/76
  - https://github.com/openebs/velero-plugin/pull/79

### CODE REVIEW

**Comment by mynktl** [comment]
Holding merge because one changelog is not part of RC1.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+v1.10.0-RC1 / 2020-05-13
```
Comment (kmova) [inline CHANGELOG.md:1]: Once RC candidates are done and a final release is made, can we also have all the notes under a major release like `v1.10.0`

Comment (mynktl) [reply CHANGELOG.md:1]: yes.

Comment (mynktl) [reply CHANGELOG.md:1]: This PR was generated by script so it is for RC1 only. I thought of creating a separate PR using script for RC1, RC2 and v1.10.0.

Comment (mynktl) [reply CHANGELOG.md]: Done. Changelog for RC2 and main release in the same PR now.

**Pull request merged**